doc/mgr/dashboard.rst

```diff
@@ -269,7 +262,7 @@ Grafana on your preferred hosts, proceed with the following steps.
     $ ceph mgr module enable prometheus
 
     More details can be found on the `documentation
-    <http://docs.ceph.com/docs/master/mgr/prometheus/>`_ of the prometheus
+    <http://docs.ceph.com/docs/luminous/mgr/prometheus/>`_ of the prometheus
```

Comment:
Not sure if we should change the release, I was actually referring to the rendering of the url:
![image](https://user-images.githubusercontent.com/399326/46483343-4f849400-c7ef-11e8-84ca-dea1cea35017.png)